src/sniffer.c

```diff
@@ -3475,17 +3474,14 @@ static int ProcessSessionTicket(const byte* input, int* sslBytes,
         session->sslServer->options.haveSessionId = 1;
 
         /* Use the wolf Session cache to retain resumption secret */
-        if (session->flags.cached == 0) {
-            WOLFSSL_SESSION* sess = wolfSSL_GetSession(session->sslServer,
-                NULL, 0);
-            if (sess == NULL) {
-                SetupSession(session->sslServer);
-                AddSession(session->sslServer); /* don't re add */
-            #ifdef WOLFSSL_SNIFFER_STATS
-                INC_STAT(SnifferStats.sslResumptionInserts);
-            #endif
-            }
-            session->flags.cached = 1;
+        WOLFSSL_SESSION* sess = wolfSSL_GetSession(session->sslServer,
```

Comment:
Can't declare a variable in the middle of a block with old C versions.
Still have customers using compilers that are old C versions.

Reply:
Fixed